inventory/debian.cf

```diff
@@ -82,6 +87,12 @@ bundle common inventory_debian
       comment => "derived from Debian",
       meta => { "inventory", "attribute_name=none" };
 
+    init_is_link::
+      "systemd"
+          expression => regcmp("/lib/systemd/systemd",
```

Comment:
Perhaps this should be ``.*/systemd$`` instead, to catch all possible paths where systemd might be installed?